packages/components/src/components/accordion/_accordion.scss

```diff
@@ -197,6 +197,25 @@
   .#{$prefix}--accordion__title.#{$prefix}--skeleton__text {
     margin-bottom: 0;
   }
+
+  .#{$prefix}--accordion--expressive {
+    overflow: hidden;
+
+    .#{$prefix}--accordion__heading {
```

Comment:
Can we unnest these selectors? We're trying to reduce nesting throughout the codebase so it's easier to search for specific selectors in the future, e.g

```scss
.#{$prefix}--accordion--expressive .#{$prefix}--accordion__heading {
    padding: $carbon--spacing-04 0;
}
```

Reply:
sure, will unnest these selectors